### PR TITLE
Auto-detect VS2015

### PR DESCRIPTION
**Pull request by jkotas**
Change build and tests scripts to detect VS2015 and use that if available by default

### CODE REVIEW

**Comment by jkotas** [comment]
@mmitche Could you please take a look?

**Comment by jkotas** [comment]
@dotnet-bot Test this please

File: build.cmd [hunk 1]
```diff
@@ -6,9 +6,11 @@ set __BuildArch=x64
 set __BuildType=Debug
 set __BuildOS=Windows_NT
 
-:: Default to VS2013
-set __VSVersion=VS2013
-set __VSProductVersion=120
+:: Default to highest Visual Studio version available
+set __VSVersion=vs2015
```
Comment (mmitche) [inline build.cmd]: The usage message below still says that the default is 2013.  Can you change that?

File: tests/buildtest.cmd [hunk 2]
```diff
@@ -11,6 +11,12 @@ set "__PackagesDir=%__ProjectDir%\packages"
 set "__RootBinDir=%__ProjectDir%\bin"
 set "__LogsDir=%__RootBinDir%\Logs"
 
+:: Default to highest Visual Studio version available
+set __VSVersion=vs2015
```
Comment (mmitche) [inline tests/buildtest.cmd]: The usage message below still says that the default is 2013.  Can you change that?

File: tests/runtest.cmd [hunk 3]
```diff
@@ -2,9 +2,11 @@
 setlocal EnableDelayedExpansion
 set __ProjectFilesDir=%~dp0
 
-:: Default to VS2013
-set __VSVersion=VS2013
-set __VSProductVersion=120
+:: Default to highest Visual Studio version available
+set __VSVersion=vs2015
```
Comment (mmitche) [inline tests/runtest.cmd]: The usage message below still says that the default is 2013.  Can you change that?

**Comment by mmitche** [comment]
@dotnet-bot test this please

**Pull request merged**